www/configurator/src/app/css-transpiler/css-transpiler.js

```diff
@@ -30,15 +32,34 @@ class CssTranspiler {
   constructor(css, cssVars) {
     this.templateCss = css;
     this.templateCssVars = cssVars;
+    /* eslint-disable */
```

Comment:
I originally did this to avoid the http://eslint.org/docs/rules/new-parens error. Since the webpack docs didn't use the () when creating new Workers. But I have noticed it does not make a different, so I will go ahead and remove this

done